packages/common/src/utils/stream-utils.ts

```diff
@@ -32,6 +32,10 @@ export class StreamUtils {
     return new StreamID(state.type, state.log[0].cid)
   }
 
+  static tipFromState(state: StreamState): CID {
+    return state.log[state.log.length - 1].cid
```

Comment:
Is there any guaranty the state log has at least 1 item please?

Reply:
Yep.  To construct a StreamState there needs to be a genesis commit